spring-remoting/remoting-http/server/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It works fine from a maven command line build on my machine too, so it's not really a problem in moving this forward. And many thanks for the link regarding that issue with Eclipse. Guess I'll have to do a fresh install of Eclipse/STS to get a clean Neon environment.